test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssert.cs

```diff
@@ -20,30 +22,62 @@ public static class XmlAssert
         /// <param name="actualXml">Actual xml string.</param>
         public static void Equal(string expectedXml, string actualXml)
         {
-            var sortedExpectedXDoc = SortAttributes(XDocument.Parse(expectedXml));
```

Comment:
that case involves two developer errors matching exactly and isn't worth worrying about.  but my comment was a nit and, by definition, doesn't require a fix if you disagree.